src/main/java/nerf/task/ToDos.java

```diff
@@ -0,0 +1,33 @@
+package nerf.task;
+
+/**
+ * Todo class
+ */
+public class ToDos extends Task{
+    public ToDos(String description) {
+        super(description);
+    }
+    public ToDos(String description, boolean status) {
```

Comment:
Coding standard violation: boolean variables should sound like booleans. Perhaps something like 

`boolean isDone` or `isComplete`